Feature/pe predefined settings

oo222bs opened the pull request:
Added a custom menu for INEL in Partitur Editor (PE) that can be activated like those for SiN, SFB538 etc (Edit > Preferences... > Menus) - it involves adding a new import dialogue for INEL which means that there will be two different Flextext Import dialogues.

berndmoos commented:
Merged and uploaded a preview for Windows and Linux. But please note: Your code is not source compatible with Java 1.6 (see the switch(String) statement in "ChooseASettingsFileDialogInel". EXMARaLDA needs to be source compatible with 1.6 because we need to build a Java 6 version for the MAC (JNI to CocoaQT in Apple's Java 6 / no video support via CocoaQT in Oracle's later versions)

berndmoos commented:
Please see above comment

oo222bs commented:
I have changed the switch block with a bunch of if/else statements. Also setformattable file was missing. I have added that as well. Moreover, there was a problem with loading the predefined setting xsl files. I have fixed that as well. Please refer to: https://github.com/EXMARaLDA/exmaralda/pull/196
